.gitignore

```diff
@@ -41,7 +41,7 @@ playground.xcworkspace
 # you should judge for yourself, the pros and cons are mentioned at:
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
-Example/Pods/
+#Example/Pods/*
```

Comment:
The reason why Sean couldn't build RCAnalytics is because downloading the project from GH does include any of the dependencies. Those would have to be installed with a `pod install` (require cocoapods.)

To support use without requiring cocoapods (which I believe we should support) requires checking all of the dependencies into source control.